com.unity.render-pipelines.high-definition/Runtime/Debug/DebugDisplay.cs

```diff
@@ -387,6 +413,12 @@ internal DebugDisplaySettings()
                 .ToArray();
             s_MsaaSamplesDebugValues = (int[])Enum.GetValues(typeof(MSAASamples));
 
+            s_RTASViewDebugStrings = Enum.GetNames(typeof(RTASDebugView)).Select(t => new GUIContent(t)).ToArray();
```

Comment:
I think you can get rid of these arrays by using DebugUI.EnumField.autoEnum.

looks like the MSAADebugValues/Strings properties are never even used. Fixing optional :)

Reply:
It's automatically generated, (RayTracing => RAY_TRACING, raytracing => RAYTRACING). Should be with a T upper case

Reply:
![image](https://user-images.githubusercontent.com/42026998/142620194-5f6daa43-0495-4f54-9131-d6028954d825.png)

Reply:
it seems to be used

Reply:
yeah sorry I meant only written to, never read.

Reply:
When you use autoEnum, you don't need to provide the enumValues and enumNames when constructing the DebugUI.EnumField. So you only want to manually pass in the values/names if you want a subset of the enum or something. But it's really optional to fix, just FYI.

Reply:
Yeah, I am too lazy to fix this one. I'll fix the other ones